README.md

```diff
@@ -207,6 +207,11 @@ See the specific [auditor docs](#auditors) for the auditor you wish to override
 
 To learn more about labels, see https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
 
+## CI/CD usage
```

Comment:
Could you add a link to this in the [table of contents](https://github.com/Shopify/kubeaudit#command-line-interface-cli) near the top of the readme?